src/clover/__main__.py

```diff
@@ -565,6 +565,7 @@ def main(  # pylint: disable=too-many-locals, too-many-statements
             finance_inputs,
             generation_inputs,
             ghg_inputs,
+             grid_attributes,
```

Comment:
## ✅ Automated tests
The :art:`black` python formatter will likely fail this line as the indentation is not in-line. Run the whole code through black and the automated tests:
* `python -m black src` to run black,
* `python -m pylint src` to run pylint etc.